source/libs/page-detect.ts

```diff
@@ -14,7 +13,7 @@ export const _is404 = domBased; // They're specified in page-detect.ts
 export const is500 = (): boolean => document.title === 'Server Error · GitHub' || document.title === 'Unicorn! · GitHub';
```

Comment:
```suggestion
export const is500 = (): boolean => document.title === 'Server Error · GitHub' || document.title === 'Unicorn! · GitHub' || document.title === '504 Gateway Time-out';
```

Github just went down and that was the error